### ISSUE TITLE
Remove hard coded mongod port

### ISSUE THREAD

**Issue by tashrifbillah** [AMP-SCZ/dpdash#9]
This port number needs to be hooked up to `configure.py` port number
https://github.com/PREDICT-DPACC/dpdash/blob/pnl-devel/singularity/setup.sh#L16

**Comment by tashrifbillah** [comment]
`--port` also needs to be passed here:
https://github.com/PREDICT-DPACC/dpdash/blob/pnl-devel/singularity/setup.sh#L17

**Comment by blms** [comment]
Those were all of the hard coded ports. In addition, I had to specify the custom port on the next few lines with the `--port` option to use a non-default port (in other words, not 27017). Using `MONGO_PORT` as an example environment variable:

```sh
mongo admin --eval "db.createUser({user: 'dpdash', pwd:'"$mongopw"', roles: [ { role: 'root', db: 'admin' } ] });" --port ${MONGO_PORT}
mongo dpdmongo --eval "db.users.insertOne({uid : 'dpdash', password:'"$adminpw"', role: 'admin', ldap: false, display_name: 'dpdash'});" --port ${MONGO_PORT}
mongod --shutdown --dbpath /data/dpdash/mongodb/dbs --port ${MONGO_PORT}
```

It may not be necessary on the `--shutdown` command.

**Comment by tashrifbillah** [comment]
We have commented at the same time!

**Comment by blms** [comment]
Closed by #14.
